debug/.gitignore

```diff
@@ -0,0 +1,4 @@
+# ./debug
+*
+
+!.gitignore
```

Comment:
This commit adds a folder "debug" that I'd rather not have in the repo. You can exclude the whole folder by changing the toplevel .gitignore.